core/object/message_queue.h

```diff
@@ -33,15 +33,15 @@
 
 #include "core/object/object_id.h"
 #include "core/os/thread_safe.h"
+#include "core/templates/local_vector.h"
 #include "core/variant/variant.h"
-
 class Object;
 
 class MessageQueue {
 	_THREAD_SAFE_CLASS_
 
 	enum {
-		DEFAULT_QUEUE_SIZE_KB = 4096
+		MAX_QUEUE_SIZE_MB = 32
```

Comment:
Since it's not really a hardcoded max but just a default value, wouldn't it be better to just set `32` in constructor instead of creating this constant?

Reply:
yeah good point